src/etl/event_processor.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, record, region):
         self._data = json.loads(self.s3.get_file(self._bucket_name, self._object_key))
         self.metadata = self._data['metadata']
         self.content = self._data['content']
+        proto_uuid = self._object_key.replace(".json", "")
+        self.uuid = proto_uuid[-36:]
```

Comment:
This could all be done in one line, but that's matter of personal preference. :man_shrugging: